OpenTelemetry.sln

```diff
@@ -222,16 +222,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "correlation", "docs\logs\co
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Tests.Stress.Logs", "test\OpenTelemetry.Tests.Stress.Logs\OpenTelemetry.Tests.Stress.Logs.csproj", "{4298057B-24E0-47B3-BB76-C17E81AF6B39}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Examples.LoggingExtensions", "examples\LoggingExtensions\Examples.LoggingExtensions.csproj", "{F5EFF065-7AF5-4D7D-8038-CC419ABD8777}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Extensions.Serilog", "src\OpenTelemetry.Extensions.Serilog\OpenTelemetry.Extensions.Serilog.csproj", "{0D85558E-15B9-4251-BDBD-9CB7933B57E2}"
```

Comment:
Why no more Serilog or EventSource extensions? Weren't they functional prototypes before, albeit with some InternalsVisibleTo hacks?

Reply:
The way the "log appender" APIs work is they have to be fed the provider to use, or get it from DI. Originally that was `OpenTelemetryLoggerProvider` and then it became `LoggerProvider`. Basically they need some of the public API we are still sorting out.

Reply:
Ok, makes sense. I'd lost track of what had been added and when.

Reply:
This actually makes me more aware of the impact of the event api scope creep. Makes me question if the logging SIG would be amenable to first spec'ing and stabilizing a LoggerProvider/GetLogger/Emit suite of APIs that did not contain any notion of event domain/name. Seems like a smaller and more well understood problem space to just support the notion of log appenders.

Reply:
Is the nuget package for OpenTelemetry.Extensions.Serilog still going to be available on myget.org, or is that going to disappear until a full solution is available?

Reply:
all things removed from main will live in a separate branch and we'll do daily myget publish from that. (Similar to what was done for Metrics in early days.)

Reply:
@pdonovan Working on that branch now it should be up soon with the changes 😄